coriolis/osmorphing/redhat.py

```diff
@@ -57,11 +59,8 @@ def __init__(self, conn, os_root_dir, os_root_dev,
         self._enable_repos = []
 
     def disable_predictable_nic_names(self):
-        kernel_versions = self._list_dir("lib/modules")
-        for version in kernel_versions:
-            cmd = '/sbin/new-kernel-pkg --update --kernel-args="%s" %s'
-            self._exec_cmd_chroot(cmd % (
-                "net.ifnames=0 biosdevname=0", version))
+        cmd = 'grubby --update-kernel=ALL --args="%s"'
```

Comment:
Will this also work on the 6s?
Even if so, we should make sure to explicitly install it to be sure it's there.

Reply:
I checked a centos6 release, it was there. Will also add an install command so we make sure it's installed.